pt-BR/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -380,58 +380,58 @@
 -->
 
 <!ENTITY showFavoritesBarCmd.accesskey "v">
-<!ENTITY pbFolderPaneSlider.label "Hide">
-<!ENTITY showFavoritesBarCmd.label "Favorites Bar">
+<!ENTITY pbFolderPaneSlider.label "Ocultar">
+<!ENTITY showFavoritesBarCmd.label "Barra de favoritos">
 
 
 <!--
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbFavoritesBarContextMenu-remove.label "Remove From Favorites">
+<!ENTITY pbFavoritesBarContextMenu-remove.label "Remover dos favoritos">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbFavoritesBarContextMenu-remove.label "Excluir dos favoritos">